app-frontend/src/app/pages/lab/browse/analyses/analyses.js

```diff
@@ -109,11 +112,39 @@ class LabBrowseAnalysesController {
         this.fetchAnalysesList(this.currentPage);
     }
 
-    onAnalysisDelete(analysisId) {
-        this.analysisService.deleteAnalysis(analysisId).then(() => {
-            this.fetchAnalysesList(this.currentPage);
+    deleteSelected() {
+        const modal = this.modalService.open({
+            component: 'rfConfirmationModal',
+            resolve: {
+                title: () => `Delete ${this.selected.size} analyses?`,
+                content: () => 'Deleting analyses will make any ' +
```

Comment:
Using back-ticks here for a multiline string would be cleaner here.

Reply:
not really. gotta have backslashes, and the next line needs to be all the way to the left to avoid extra spaces in the string.